### ISSUE TITLE
Aggregator discovery

### ISSUE THREAD

**Issue by chris-wood** [abetterinternet/prio-documents#9]
Clients need to obtain aggregator public keys to encrypt shares. While we may not want to specify the exact mechanism by which deployments do this, we should minimally state the requirements for discovery, perhaps with a RECOMMENDED version.

cc @ekr

**Comment by tgeoghegan** [comment]
Some quick thoughts: 

1. Thus far we have talked about clients discovering and trusting aggregators. There is a related problem of aggregators discovering each other (e.g. in order to exchange validations, or just for each aggregator to discover the leader), which can be probably be solved the same way.
2. We didn't do an amazing job of client<->aggregator trust in Prio v2 (mostly because no matter how you sliced it, we had to trust that Apple and Google's client implementations weren't just sending data to Cupertino or Mountain View in the clear anyway), but we addressed aggregator<->aggregator and aggregator<->collector trust by bootstrapping from the web PKI: each participating server vended configuration parameters (including crypto keys) from a JSON file vended over TLS. Participating organizations agreed out of band on what trusted domains for config should be (i.e., `isrg-prio.org` for us, `en-analytics.cancer.gov` for the feds, `apple.com/exposure-notifications`, and so on). So one could imagine the leader in the Prio v3 scheme serving a directory of aggregators, which would be a list of URLs from which the config for each server could then be fetched and validated, independently of the client's trust relationship with the leader.
